src/components/screens/Service.tsx

```diff
@@ -123,9 +128,20 @@ export const Service = ({
     </DiagonalGradient>
   ) : (
     <>
-      <WrapperWrap spaceAround={shouldCenter} spaceBetween={!shouldCenter}>
-        {tiles?.map(renderItem)}
-      </WrapperWrap>
+      {rows.map((row, index) => {
+        const isLastRow = index === rows.length - 1;
+        const isIncompleteRow = row.length < itemsPerRow;
+
+        return (
+          <WrapperWrap
+            key={index}
```

Comment:
we learned somwhen, that the index ist not the best choice for a key. please think of something more unique here.

Reply:
I updated the key with `tile.title || accessibilityLabel`.